app/dao/notifications_dao.py

```diff
@@ -751,6 +751,16 @@ def dao_get_total_notifications_sent_per_day_for_performance_platform(start_date
     )
 
 
+@statsd(namespace="dao")
+def get_latest_sent_notification_for_job(job_id):
+    return (
+        Notification.query.filter(Notification.job_id == job_id, Notification.status == "sent")
```

Comment:
I think we don't need to filter on `Notification.status == "sent"` here. If the notification has been added to the db then that is good enough

Reply:
Fixed!